lib/index.js

```diff
@@ -72,7 +72,7 @@ controller.on('interactive_message_callback', (bot, message) => {
     bot.replyInteractive(message, {
       attachments: [{
         ...replyMessages.shouldSubmitAttachment,
-        text: submitRequest ? replyMessages.submitAnswer : replyMessages.doNotSubmitAnswer,
+        text: submitRequest ? replyMessages.requestSent : replyMessages.doNotSubmitAnswer,
```

Comment:
actually I think this `submitRequest` will only be false at this point, since its inside of that condition, I will appreciate if you could do that update.